.changeset/metal-cups-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/mongo-nav': minor
```

Comment:
I think we should make this a major version bump as we're changing the UserMenu across products, what do you think cc: @DesignerDave

Reply:
I don't feel strongly either way. This isn't breaking from an API standpoint, so the argument would be simply that the design change is large enough to warrant it, which I don't feel strongly that it is or isn't. Your call 😄

Reply:
I feel better about this being a minor version, now that the determination re: whether to render the banner/or not is coming from the server rather than from us imposing it on every product